Modix.Services/Core/UserService.cs

```diff
@@ -145,6 +145,9 @@ public async Task<GuildUserSummary> GetGuildUserSummaryAsync(ulong guildId, ulon
         /// <inheritdoc />
         public async Task<EphemeralUser> GetUserInformationAsync(ulong guildId, ulong userId)
         {
+            if (userId <= 0)
```

Comment:
A ulong can never be less than 0

Reply:
Good point

Reply:
Had to resort to being extremely picky as I couldn't find any other issues :^)